element_facemap/facial_behavior_estimation.py

```diff
@@ -234,16 +231,11 @@ def make(self, key):
 
             video_files = (FacemapTask * VideoRecording.File & key).fetch('file_path')
             video_files = [find_full_path(get_facemap_root_data_dir(), video_file).as_posix() for video_file in video_files]
-            print(video_files)
 
             output_dir = find_full_path(get_facemap_root_data_dir(), output_dir)
-            
-            # facemap_params = np.load('/Volumes/DincerDJ/workflow_facemap_testset/subject0/session0/facevideo1_proc.npy', allow_pickle=True).item()
-
-            facemap_run([video_files], proc=facemap_params, savepath=output_dir.as_posix())
+            facemap_run([video_files], sbin=facemap_params['sbin'], proc=facemap_params, savepath=output_dir.as_posix(), motSVD=facemap_params['motSVD'], movSVD=facemap_params['movSVD'])
```

Comment:
Dimitri recommended I download a linter for PEP8 line length issues. Not crucial but helps readability
```suggestion
            facemap_run([video_files], sbin=facemap_params['sbin'], proc=facemap_params,
                        savepath=output_dir.as_posix(), motSVD=facemap_params['motSVD'],
                        movSVD=facemap_params['movSVD'])
```

Reply:
Thank you for the suggestions! It’s  on my to do list now.

Reply:
Looks like 88 is not long enough to explain some of the table attributes.